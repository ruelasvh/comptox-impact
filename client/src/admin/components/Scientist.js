/**
 * Created by Victor H. Ruelas-Rivera on 6/14/17.
 * US EPA National Center for Computational Toxicology
 */
import React from 'react';
import ScintistProfile from '../../scientist/ScientistProfile';

class Scientist extends React.Component {
    render() {
        return (
            <div>
                <ScintistProfile {...this.props}/>
            </div>
        )
    }
}

export default Scientist;