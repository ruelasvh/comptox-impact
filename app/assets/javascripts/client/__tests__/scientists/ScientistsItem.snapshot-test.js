/**
 * Created by Victor H. Ruelas-Rivera on 2/1/17.
 * US EPA National Center for Computational Toxicology
 */
import React from 'react';
import Scientist from '../../src/scientists/components/ScientistsIndexItem';
import renderer from 'react-test-renderer';
import seed from '../../mocks/scientistMock';

test('ScientistsItem renders correctly', () => {
    const component = renderer.create(
        <Scientist scientist={seed} scientists_path="/scientists" i={1}/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});