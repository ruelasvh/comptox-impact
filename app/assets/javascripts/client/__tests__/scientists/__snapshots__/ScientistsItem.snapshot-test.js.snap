exports[`test ScientistsItem renders correctly 1`] = `
<div
  className="scientists-item">
  <a
    onClick={[Function]}
    style={Object {}}>
    <div
      className="thumbnail">
      <img
        alt="Scientist_Image"
        src="test-file-stub" />
      <div
        className="caption">
        <h4>
          Juan 
          <span
            className="last-name">
            Perez
          </span>
        </h4>
        <p>
          Rocket Scientist
        </p>
      </div>
    </div>
  </a>
</div>
`;
